factory/status/statusComponents.go

```diff
@@ -64,6 +66,7 @@ type statusComponentsFactory struct {
 	cryptoComponents     factory.CryptoComponentsHolder
 	isInImportMode       bool
 	isSovereign          bool
+	esdtPrefix           string
```

Comment:
Why Pass `esdPrefix` all the way down to here? Don't you already have the whole config in `statusComponentsFactory`?

Reply:
It's just the `GeneralConfig`, and I needed `SystemSCConfig` for the prefix.